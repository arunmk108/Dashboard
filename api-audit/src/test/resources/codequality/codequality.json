[
  {
    "id" : "5b76eb7241208e0930c7870c",
    "collectorItemId" : "56e5919d7fab7c0244e33341",
    "timestamp" : 1476472229000,
    "name" : "hubble-ui",
    "url" : "",
    "type" : "StaticAnalysis",
    "version" : "1.0",
    "metrics" : [
      {
        "name" : "blocker_violations",
        "value" : "0",
        "formattedValue" : "0"
      },
      {
        "name" : "coverage",
        "value" : "93.5",
        "formattedValue" : "93.5%"
      },
      {
        "name" : "alert_status",
        "value" : "WARN",
        "formattedValue" : "WARN"
      },
      {
        "name" : "major_violations",
        "value" : "0",
        "formattedValue" : "0"
      },
      {
        "name" : "new_vulnerabilities"
      },
      {
        "name" : "ncloc",
        "value" : "1447",
        "formattedValue" : "1,447"
      },
      {
        "name" : "violations",
        "value" : "65",
        "formattedValue" : "65"
      },
      {
        "name" : "line_coverage",
        "value" : "96.3",
        "formattedValue" : "96.3%"
      },
      {
        "name" : "quality_gate_details",
        "value" : "{\"level\":\"WARN\",\"conditions\":[{\"metric\":\"blocker_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"critical_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"major_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"minor_violations\",\"op\":\"GT\",\"warning\":\"60\",\"error\":\"70\",\"actual\":\"65\",\"level\":\"WARN\"},{\"metric\":\"coverage\",\"op\":\"LT\",\"warning\":\"95\",\"error\":\"90\",\"actual\":\"93.5\",\"level\":\"WARN\"},{\"metric\":\"line_coverage\",\"op\":\"LT\",\"warning\":\"95\",\"error\":\"90\",\"actual\":\"96.3\",\"level\":\"OK\"}],\"ignoredConditions\":false}",
        "formattedValue" : "{\"level\":\"WARN\",\"conditions\":[{\"metric\":\"blocker_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"critical_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"major_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"minor_violations\",\"op\":\"GT\",\"warning\":\"60\",\"error\":\"70\",\"actual\":\"65\",\"level\":\"WARN\"},{\"metric\":\"coverage\",\"op\":\"LT\",\"warning\":\"95\",\"error\":\"90\",\"actual\":\"93.5\",\"level\":\"WARN\"},{\"metric\":\"line_coverage\",\"op\":\"LT\",\"warning\":\"95\",\"error\":\"90\",\"actual\":\"96.3\",\"level\":\"OK\"}],\"ignoredConditions\":false}%"
      },
      {
        "name" : "sqale_index",
        "value" : "298",
        "formattedValue" : "4h 58min"
      },
      {
        "name" : "critical_violations",
        "value" : "0",
        "formattedValue" : "0"
      }
    ]
  }

]