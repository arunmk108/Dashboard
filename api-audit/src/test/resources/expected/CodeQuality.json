{
  "auditStatuses": [
    "DASHBOARD_CODEQUALITY_CONFIGURED"
  ],
  "lastUpdated": 0,
  "dashboardTitle": "TestSSA",
  "businessService": "TestBusServ",
  "businessApplication": "confItem",
  "review": {
    "CODE_QUALITY": [
      {
        "auditStatuses": [
          "CODE_QUALITY_THRESHOLD_BLOCKER_MET",
          "CODE_QUALITY_THRESHOLD_CRITICAL_FOUND",
          "QUALITY_PROFILE_VALIDATION_AUDIT_NO_CHANGE",
          "CODE_QUALITY_THRESHOLD_CODE_COVERAGE_FOUND",
          "CODE_QUALITY_AUDIT_FAIL",
          "CODE_QUALITY_THRESHOLD_CRITICAL_MET",
          "CODE_QUALITY_GATES_FOUND",
          "CODE_QUALITY_THRESHOLD_BLOCKER_FOUND",
          "CODE_QUALITY_CHECK_IS_CURRENT"
        ],
        "lastUpdated": 0,
        "url": "",
        "lastExecutionTime": 1476472229000,
        "codeQuality": {
          "id": "5b76eb7241208e0930c7870c",
          "collectorItemId": "56e5919d7fab7c0244e33341",
          "timestamp": 1476472229000,
          "name": "hubble-ui",
          "url": "",
          "type": "StaticAnalysis",
          "version": "1.0",
          "metrics": [
            {
              "name": "blocker_violations",
              "value": "0",
              "formattedValue": "0"
            },
            {
              "name": "coverage",
              "value": "93.5",
              "formattedValue": "93.5%"
            },
            {
              "name": "alert_status",
              "value": "WARN",
              "formattedValue": "WARN"
            },
            {
              "name": "major_violations",
              "value": "0",
              "formattedValue": "0"
            },
            {
              "name": "new_vulnerabilities"
            },
            {
              "name": "ncloc",
              "value": "1447",
              "formattedValue": "1,447"
            },
            {
              "name": "violations",
              "value": "65",
              "formattedValue": "65"
            },
            {
              "name": "line_coverage",
              "value": "96.3",
              "formattedValue": "96.3%"
            },
            {
              "name": "quality_gate_details",
              "value": "{\"level\":\"WARN\",\"conditions\":[{\"metric\":\"blocker_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"critical_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"major_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"minor_violations\",\"op\":\"GT\",\"warning\":\"60\",\"error\":\"70\",\"actual\":\"65\",\"level\":\"WARN\"},{\"metric\":\"coverage\",\"op\":\"LT\",\"warning\":\"95\",\"error\":\"90\",\"actual\":\"93.5\",\"level\":\"WARN\"},{\"metric\":\"line_coverage\",\"op\":\"LT\",\"warning\":\"95\",\"error\":\"90\",\"actual\":\"96.3\",\"level\":\"OK\"}],\"ignoredConditions\":false}",
              "formattedValue": "{\"level\":\"WARN\",\"conditions\":[{\"metric\":\"blocker_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"critical_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"major_violations\",\"op\":\"GT\",\"warning\":\"\",\"error\":\"0\",\"actual\":\"0\",\"level\":\"OK\"},{\"metric\":\"minor_violations\",\"op\":\"GT\",\"warning\":\"60\",\"error\":\"70\",\"actual\":\"65\",\"level\":\"WARN\"},{\"metric\":\"coverage\",\"op\":\"LT\",\"warning\":\"95\",\"error\":\"90\",\"actual\":\"93.5\",\"level\":\"WARN\"},{\"metric\":\"line_coverage\",\"op\":\"LT\",\"warning\":\"95\",\"error\":\"90\",\"actual\":\"96.3\",\"level\":\"OK\"}],\"ignoredConditions\":false}%"
            },
            {
              "name": "sqale_index",
              "value": "298",
              "formattedValue": "4h 58min"
            },
            {
              "name": "critical_violations",
              "value": "0",
              "formattedValue": "0"
            }
          ]
        },
        "codeAuthors": [],
        "configChanges": []
      }
    ]
  }
}